docs/API_DESIGN.md

```diff
@@ -90,11 +93,10 @@ CLASS VerifiableCredential
   PUBLIC DATA credentialSubject: CredentialSubject
 
   CONSTRUCTOR create(issuer: Issuer, credential_subject: CredentialSubject, options: CreateOptions)
+  CONSTRUCTOR from_vc_jwt(vc_jwt: string, verify: bool)
```

Comment:
@nitro-neal @diehuxx I forgot about this yesterday during our sync... do we want to enable developers to bring-their-own `Verifier`? If so how would that fit into this function signature?

Reply:
One more item @nitro-neal @diehuxx, regardless of the above comment, should the default behavior be for cryptographic verification to occur? So with the code as it is currently that would mean `verify: bool = true` (though if that were the case I think we should stick with `bool` values always defaulting to false so we would change the name to `disable_verification: bool`).

Reply:
We chatted offline, so I'm summarizing our discussion here to document.

IMO, BYO verifier is extraneous. If someone wants to use their own verifier, they can use it after decoding. Passing a custom verifier into the decoder is added complexity in our API and implementation.